tests/www/approvals_views/test_prolongation_requests.py

```diff
@@ -136,6 +145,19 @@ def test_show_view(snapshot, client):
     assert_previous_step(response, reverse("approvals:prolongation_requests_list") + "?only_pending=on")
 
 
+@pytest.mark.ignore_unknown_variable_template_error("hiring_pending", "job_application", "matomo_event_attrs")
+def test_show_view_no_siae(client):
+    prolongation_request = approvals_factories.ProlongationRequestFactory(for_snapshot=True)
```

Comment:
Vu que tu n'utilises pas de snapshot dans le test autant "simplifier" je pense.
```suggestion
    prolongation_request = approvals_factories.ProlongationRequestFactory()
```